app/src/components/InputElement/InputElement.styles.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+import background from "../../images/bg-shorten-desktop.svg";
+
+export const StyledForm = styled.form`
+  position: absolute;
+  left: 270px;
+  border: 1px solid black;
```

Comment:
"black" should be taken from the theme colors

Reply:
already fixed